### PR TITLE
[ART-8482] Add labels for better JIRA tracking

### PR DESCRIPTION
**Pull request by ashwindasr**
ref https://issues.redhat.com/browse/ART-8482

### CODE REVIEW

File: doozer/doozerlib/cli/images_streams.py [hunk 1]
```diff
@@ -796,6 +796,7 @@ def search_issues(query):
         fields = {
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
+            'labels': ["art:reconciliation", f"art:distgit:{distgit_key}"],
```
Comment (sosiouxme) [inline doozer/doozerlib/cli/images_streams.py:799]: this is petty but good style is to use consistent quoting
```suggestion
            'labels': ['art:reconciliation', f'art:distgit:{distgit_key}'],
```
(good style is also to use double quotes, not single, but that's more controversial)

**Review by sosiouxme — COMMENTED** [review]
/lgtm

File: doozer/doozerlib/cli/images_streams.py [hunk 2]
```diff
@@ -796,6 +796,7 @@ def search_issues(query):
         fields = {
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
+            'labels': ['art:reconciliation', f'art:distgit:{distgit_key}'],
```
Comment (joepvd) [inline doozer/doozerlib/cli/images_streams.py:799]: I think `imagemeta` should have `get_component_name()`, which returns the brew component. Think there's some value in uniformity here.

Comment (ashwindasr) [reply doozer/doozerlib/cli/images_streams.py]: I made a wrong assumption that get_component_name retrieves the component in product.yaml page.

Updated

**Review by joepvd — APPROVED** [review]
/lgtm

**Pull request merged**